backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell
+
+import com.typesafe.config.Config
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import wdl4s.{CallInputs, NamespaceWithWorkflow, Task, WdlExpression}
+
+package object backend {
+
+  trait BackendJobDescriptorKey {
+    def callFqn: String
+    def index: Option[Int]
+    def attempt: Int
+  }
+
+  // Can become a case class once Khalid's work simplifies the world so all calls run on the backend.
```

Comment:
actually this lives in `cromwell.backend` and is not the `JobDescriptor` used in the rest of the Cromwell codebase, so this case-classification could happen now